src/main/webapp/vendor/startbootstrap/planner.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
```

Comment:
i think this file can be deleted now right, if we're going to render it straight from jsp?